packages/fxa-auth-server/config/index.ts

```diff
@@ -402,6 +402,12 @@ const conf = convict({
       format: 'port',
       doc: 'Port for Redis server',
     },
+    connectTimeout: {
```

Comment:
@dannycoates Is this something worthwhile? The CI timeout errors looks like this https://app.circleci.com/pipelines/github/mozilla/fxa/12046/workflows/81796a6d-794e-447f-8595-a490df59a73e/jobs/181342/steps

Reply:
that's a mysql timeout

Reply:
Hrm, that one is indeed mysql. Let me see if I can reproduce the error I got locally with the error stack.

Do the oauth server tests pass for you locally?